src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -3,33 +3,150 @@
  * @author Box
  */
 import * as React from 'react';
-import flow from 'lodash/flow';
 import { FormattedMessage } from 'react-intl';
+import flow from 'lodash/flow';
+import getProp from 'lodash/get';
+import { MetadataEmptyState } from '@box/metadata-editor';
+import type { MessageDescriptor } from 'react-intl';
+import messages from '../common/messages';
+import { normalizeTemplates } from '../../features/metadata-instance-editor/metadataUtil';
+import { isUserCorrectableError } from '../../utils/error';
 import { withAPIContext } from '../common/api-context';
 import { withErrorBoundary } from '../common/error-boundary';
 import { withLogger } from '../common/logger';
-import { ORIGIN_METADATA_SIDEBAR_REDESIGN } from '../../constants';
-import { EVENT_JS_READY } from '../common/logger/constants';
 import { mark } from '../../utils/performance';
-import messages from '../common/messages';
+import { EVENT_JS_READY } from '../common/logger/constants';
+import API from '../../api';
+import {
+    FIELD_IS_EXTERNALLY_OWNED,
+    FIELD_PERMISSIONS,
+    FIELD_PERMISSIONS_CAN_UPLOAD,
+    IS_ERROR_DISPLAYED,
+    ORIGIN_METADATA_SIDEBAR_REDESIGN,
+} from '../../constants';
+import type { ElementsXhrError, ErrorContextProps } from '../../common/types/api';
+import type { BoxItem } from '../../common/types/core';
+import type { MetadataEditor, MetadataTemplate } from '../../common/types/metadata';
+import type { WithLoggerProps } from '../../common/types/logging';
 
 import './MetadataSidebarRedesign.scss';
 
+interface ExternalProps {
+    isBoxAiSuggestionsFeatureEnabled: boolean;
+    isFeatureEnabled: boolean;
```

Comment:
Is this flag still necessary? I noticed its set to `true` in the original by default but is it possible to be on the metadata tab but having the feature set to false?

Reply:
I wonder the same thing. If you look at https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/SidebarPanels.js#L280 you'll see that there is already and enabled check for this component. That said, this field will be passed down as it's not optional and we are sending it to the backend metadata api call so I think we do still need it.